### PR TITLE
Add Getting Started steps to README

### PR DESCRIPTION
**Pull request by mefryar**
## 🤔 Problem
In looking to get started with `clojure-repl-intellij`, the first place I checked was the `README`. Currently, the `README` does not have Getting Started instructions.

## 🧐 Solution
Add a modified version of the Getting Started steps from the [IntelliJ Plugin Marketplace page](https://plugins.jetbrains.com/plugin/23073-clojure-repl) to the `README`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Checkout all available [features](https://github.com/afucher/clojure-repl-intell
 ![Clojure LSP Intellij](images/demo.png)
 
 ---
+## Getting Started
+After installing the plugin in IntelliJ, you can add a REPL to your Run
+configurations.
+1. Go to `Run` > `Edit Configurations`
+2. Click `Add new` > `Clojure REPL` > `Remote`
```
Comment (ericdallo) [inline README.md:23]: shouldn't we mention about local repl as well besides the remote one?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ Checkout all available [features](https://github.com/afucher/clojure-repl-intell
 ![Clojure LSP Intellij](images/demo.png)
 
 ---
+## Getting Started
```
Comment (ericdallo) [inline README.md:19]: will this already work to show in the plugin getting started section from intellij? I recall intellij docs mentioning how to do that but can't remember how

File: README.md [hunk 3]
```diff
@@ -16,6 +16,12 @@ Checkout all available [features](https://github.com/afucher/clojure-repl-intell
...
+After installing the plugin in IntelliJ, you can add a REPL to your Run
+configurations.
+1. Go to `Run` > `Edit Configurations`
+2. Click `Add new` > `Clojure REPL` > `Remote`
+3. Copy the values of host and port from an existing nREPL process
```
Comment (ericdallo) [inline README.md:24]:
```suggestion
3. Copy the values of host and port from an existing nREPL process, paste and save the configuration
```

File: README.md (hunk 2 above)
Comment (afucher) [reply README.md:19]: No, unfortunately, I need to do it manually on the admin page.
But once the PR is merged, I can update it there.

File: README.md (hunk 1 above)
Comment (afucher) [reply README.md:23]: agree @ericdallo. I think that we can have two sections inside Getting Started, Local and Remote.

File: README.md (hunk 3 above)
Comment (mefryar) [reply README.md]: Thanks for the suggestion! Instructions updated with [871d445](https://github.com/afucher/clojure-repl-intellij/pull/43/commits/871d445247d24db77383e549009fa1ede6bde872).

File: README.md (hunk 1 above)
Comment (mefryar) [reply README.md]: Thanks for flagging! I saw "Start a nREPL server from IntelliJ" listed under the "Soon" section in the `README` so I assumed it wasn't yet possible. I just tested and confirmed that it works! With [871d445](https://github.com/afucher/clojure-repl-intellij/pull/43/commits/871d445247d24db77383e549009fa1ede6bde872), I've added separate instructions for Local and Remote, as well as moved "Start a nREPL server from IntelliJ" from "Soon" to "Features".

**Review by ericdallo — APPROVED** [review]

**Comment by ericdallo** [comment]
@afucher can you merge and update the Jetbrains marketplace?

**Pull request merged**